src/scheduled_tasks.rs

```diff
@@ -111,49 +141,44 @@ pub async fn setup(context: Data<LemmyContext>) -> LemmyResult<()> {
 }
```

Comment:
Changed this file so that all the individual tasks return result, and only the top-level handler logs them. That way we can use a simple unit test to ensure that no errors are thrown.